src/components/common/applicant-info-1.vue

```diff
@@ -594,6 +594,10 @@ export default class ApplicantInfo1 extends Mixins(ActionMixin) {
     this.showAddressMenu = false
   }
 
+  mutateNRData (key, value) {
```

Comment:
Why "mutate" instead of "update" line elsewhere. It sounds like it belongs in a store module...?

Reply:
Yea, it was a carry over from the original. I will update the local method name to `updateNrData`.